### PR TITLE
Hailem/110 refactor the card component

### LINKED ISSUE #110: Refactor the Card component
- [ ] Rename CardList component as ResultList
- [ ] Rename Card component as ResultElement

### PR DESCRIPTION
**Pull request by hailem**
fixes #110

### CODE REVIEW

File: packages/cards/src/card.ts [hunk 1]
```diff
@@ -5,22 +5,22 @@ import { GenericObject, EventName, TemplateFunction, Widget, template } from '@d
 import { STYLES } from './styles/card';
 
 const DEFAULT_CLICK_DURATION = 200;
-const DEFAULT_TEMPLATE = function(html: Function, ctx: GenericObject) {
+const DEFAULT_TEMPLATE = function(html: Function) {
```
Comment (carlosescri) [inline packages/cards/src/card.ts]: Don't change the signature of the function. ALL template functions MUST use the same signature. If TS complains, use:

```ts
function(html: Function, _ctx: GenericObject)
```

To mark the parameter as unused.
```

File: playground/components/cards.html [hunk 2]
```diff
@@ -34,7 +34,7 @@ <h2>Discrete Results</h2>
       </df-input-capture>
       <df-total></df-total>
     </div>
-    <df-card-list currency="EUR"></df-card-list currency="EUR">
+    <df-results currency="EUR"></df-results currency="EUR">
```
Comment (carlosescri) [inline playground/components/cards.html]: Fix the typo (vscode's fault, I didn't noticed the typo when I commited it), should be:

```html
<df-results currency="EUR"></df-results>
```

File: packages/cards/src/card.ts (hunk 1 above)
Comment (hailem) [reply packages/cards/src/card.ts]: done

File: playground/components/cards.html (hunk 2 above)
Comment (hailem) [reply playground/components/cards.html]: done

**Pull request merged**